builder-18.toml

```diff
@@ -61,12 +61,12 @@ version = "0.10.1"
   uri = "https://github.com/forcedotcom/nodejs-sf-fx-buildpack/releases/download/v2.0.6/nodejs-sf-fx-buildpack-v2.0.6.tgz"
 
 [[buildpacks]]
-  id = "projectriff/streaming-http-adapter"
-  uri = "docker://gcr.io/projectriff/streaming-http-adapter:1.4.0"
+  id = "heroku/streaming-http-adapter-buildpack"
+  uri = "https://heroku-build-artifacts.s3.us-east-2.amazonaws.com/streaming-http-adapter-buildpack/streaming-http-adapter-linux-amd64-latest.tgz"
```

Comment:
Previously this URI was pointing to a buildpack, now it's pointing to a raw binary.